src/utils/components/ListPageFilter/ListPageFilter.tsx

```diff
@@ -186,6 +204,26 @@ const ListPageFilter: FC<ListPageFilterProps> = ({ data, loaded, onFilterChange,
             </ToolbarFilter>
           </ToolbarItem>
         </ToolbarToggleGroup>
+        {columnLayout?.id && !hideColumnManagement && (
```

Comment:
I'd move this component to a seprate file as this file is already over 200 lines